packages/perseus/src/validation.types.ts

```diff
@@ -236,58 +238,105 @@ export type PerseusTableRubric = {
 
 export type PerseusTableUserInput = ReadonlyArray<ReadonlyArray<string>>;
 
-export type Rubric =
-    | PerseusCategorizerScoringData
-    | PerseusCSProgramRubric
-    | PerseusDropdownRubric
-    | PerseusExpressionRubric
-    | PerseusGroupRubric
-    | PerseusGradedGroupRubric
-    | PerseusGradedGroupSetRubric
-    | PerseusGrapherRubric
-    | PerseusIFrameRubric
-    | PerseusInputNumberRubric
-    | PerseusInteractiveGraphRubric
-    | PerseusLabelImageRubric
-    | PerseusMatcherRubric
-    | PerseusMatrixRubric
-    | PerseusNumberLineScoringData
-    | PerseusNumericInputRubric
-    | PerseusOrdererRubric
-    | PerseusPlotterScoringData
-    | PerseusRadioRubric
-    | PerseusSorterRubric
-    | PerseusTableRubric;
-
-export type UserInput =
```

Comment:
This is moved down below all of the Rubric/Scoring Data types together with the various widget UserInput types. #organizing